# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acroread/setup.diff
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- INSTALL.orig	2006-11-29 12:59:33.000000000 +0100
+++ INSTALL	2006-11-29 13:06:42.000000000 +0100
@@ -679,8 +679,12 @@
 UpdateIcon()
 {
     if [ -n "$1" -a -n "$2" ]; then
-        rm -f "$2" 2>/dev/null
-        ln -s "$1" "$2" 2>/dev/null
+        if [ -e "$2" ]; then
+          ln -s "$1" "$2.acrobat" 2>/dev/null
+        else
+          #rm -f "$2" 2>/dev/null
+          ln -s "$1" "$2" 2>/dev/null
+        fi
     fi
 }
 
@@ -857,7 +861,7 @@
       OLD_IFS="$IFS"
       IFS=':'
 
-      for idir in $PATH; do
+      for idir in $DUMMY; do
         if [ -x "${idir}"/update-mime-database ]; then
             "${idir}"/update-mime-database "$GNOME_BASE_DIR/share/mime" >/dev/null 2>&1
         fi
